src/renderer/base/renderer.cpp

```diff
@@ -62,12 +64,7 @@ Renderer::~Renderer()
 // - HRESULT S_OK, GDI error, Safe Math error, or state/argument errors.
 [[nodiscard]] HRESULT Renderer::PaintFrame()
 {
-    if (_destructing)
-    {
-        return S_FALSE;
-    }

```

Comment:
Did `_destructing` make a comeback during this PR, but this block didn't?

Reply:
We already got the check in the beginning of the loop, so I figured we don't need it outside of it as well.

Reply:
oh derp yea of course we don't